vms/volunteer/tests.py

```diff
@@ -1,54 +1,188 @@
-import datetime
 from django.contrib.auth.models import User
 from django.test import TestCase
+
 from organization.models import Organization
 from volunteer.models import Volunteer
-from volunteer.services import *
+from volunteer.services import delete_volunteer, delete_volunteer_resume, get_all_volunteers, get_volunteer_by_id, get_volunteer_resume_file_url, get_volunteers_ordered_by_first_name, has_resume_file, search_volunteers
```

Comment:
Nice! If you wish to fix the line length here. You could put in parens (delete_volunteer, .... search_volunteers), then you can break the line